renovate.json

```diff
@@ -15,6 +15,12 @@
     {
       "matchPackageNames": ["node", "npm", "@tinymce/tinymce-react"],
       "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["@edx"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "automerge": true,
+      "automergeSchedule": ["at any time"]
```

Comment:
Is it okay to schedule it to anytime? (please enlighten me since I may not have full context on this.)

Reply:
this will merge the PR of `@edx` minor and patch package upgrades anytime and automatically. Mostly the minor and patch upgrades doesn't have breaking changes so we can safely merge it anytime.